### DISCUSSION TITLE
Resizeable in Gtk

### DISCUSSION THREAD

**Discussion by DavidMcGaha** [picoe/Eto#1933] [Q&A] [answered]
I have been trying to get resizeable=false to work in Gtk both Windows and Linux. I have been unsuccessful. It works in WPF. Am I needing set a complimentary setting. I also tested a vanilla Gtk# app and the resizeable flag works as expected. Is anyone else experiencing this?

**Comment by cwensley** [comment, marked as answer]
Yes this was broken in Gtk due to issues with using Gtk's Resizable flag which makes it ignore any sizing you have set.  The latest in develop build _should_ fix this however after #1930

**Reply by DavidMcGaha** [reply]
Thanks Curtis
